__tests__/app.test.js

```diff
@@ -35,7 +35,7 @@ describe("GET /api/reviews/:review_id", () => {
         .expect(200)
         .then((response) => {
             const output = response.body.review
-            expect(Object.keys(output).length).toBe(9)
+            expect(Object.keys(output).length).toBe(10)
```

Comment:
We should not have to amend a previous test for new functionality. What if you needed to add more keys to the review object?